corehq/apps/export/models/new.py

```diff
@@ -118,6 +121,23 @@ class TableConfiguration(DocumentSchema):
     columns = ListProperty(ExportColumn)
     selected = BooleanProperty(default=False)
 
+    @property
+    def identifier(self):
```

Comment:
Ah right of course. Fixed here: https://github.com/dimagi/commcare-hq/commit/aa956092271baa67ed7cafecaefbc82ace76481c